python/core/auto_generated/qgis.sip.in

```diff
@@ -1589,6 +1589,15 @@ The development version
       Unknown,
     };
 
+    enum class ScriptLanguageCapability
+    {
+      Reformat,
+      CheckSyntax,
+    };
```

Comment:
Thoughts about adding `ToggleComment` here? It is a "local" action though, not a global one.

Reply:
That's a good idea!

Reply:
Done